gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -162,59 +173,84 @@ end
 -- @return table
 -- @realm client
 function LANG.GetUnsafeLanguageTable()
-	return cached_active
+	return cachedActive
+end
+
+---
+-- Caused by a recent rename of the language names, old addons loos compatibility.
+-- To prevent annoyance, a compatibiliy mode is introduced.
+-- @param string langName string key The identifier for the language
+-- @return string The lang name in the new format
```

Comment:
```suggestion
---
-- Caused by a recent rename of the language names, old addons lost compatibility.
-- To prevent annoyance, a compatibility mode is introduced.
-- @param string langName The identifier for the language
-- @return string The language name in the new format
```